### DISCUSSION TITLE
Create two instances of the same version in different directories.

### DISCUSSION THREAD

**Discussion by badprogrammer42** [headlesshq/headlessmc#194] [Q&A]
Hey,
I would like to ask, if it is possible to create two instances of the same version (e.g. two Forge 1.8.9 Instances) each in a different game directory with Headless MC?

**Comment by 3arthqu4ke** [comment]
Yeah many ways to do this, start headlessmc twice with two different configs, or just configure the gameDir property for each instance etc. etc.

**Reply by badprogrammer42** [reply]
thank you for your quick response,
can you please tell me how I change the game dir of an instance in headlessmc? Maybe its obvious and I just don't see it but I don't quite get how I do this.

**Reply by 3arthqu4ke** [reply]
You can put `hmc.gamedir=<path to Game dir>` into your config.properties, or you start headlessmc with the `-Dhmc.gamedir=...` system property.

**Reply by badprogrammer42** [reply]
Ok, thank you for your help. Worked perfectly :)
